arangosh/Dump/DumpFeature.cpp

```diff
@@ -1041,13 +1041,19 @@ void DumpFeature::start() {
 
   // check if we are in cluster or single-server mode
   Result result{TRI_ERROR_NO_ERROR};
-  std::tie(result, _options.clusterMode) = _clientManager.getArangoIsCluster(*httpClient);
+  std::string role;
+  std::tie(result, role) = _clientManager.getArangoIsCluster(*httpClient);
+  _options.clusterMode = (role == "COORDINATOR");
   if (result.fail()) {
-    LOG_TOPIC(FATAL, arangodb::Logger::RESTORE)
+    LOG_TOPIC(FATAL, arangodb::Logger::DUMP)
         << "Error: could not detect ArangoDB instance type: " << result.errorMessage();
     FATAL_ERROR_EXIT();
   }
 
+  if (role == "DBSERVER" || role == "PRIMARY") {
+    LOG_TOPIC(WARN, arangodb::Logger::DUMP) << "operations in the cluster should be carried via a coordinator, but we are connected to a database server node. this is an unsupported operation!";
```

Comment:
Is it really `carried` or should it be `carried out`? Maybe rephrase a little? CC @sleto-it 

```suggestion
    LOG_TOPIC(WARN, arangodb::Logger::DUMP) << "You connected to a DBServer node, but operations in a cluster should be carried out via a Coordinator. This is an unsupported operation!";
```